recipes/resiprocate/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,50 @@
+#include <resip/stack/SipMessage.hxx>
+#include <resip/stack/Helper.hxx>
+#include <rutil/Logger.hxx>
+#include <fstream>
+#include <iostream>
+
+#define RESIPROCATE_SUBSYSTEM Subsystem::TEST
+
+using namespace resip;
+
+int main(int argc, char* argv[])
+{    
+    if (argc < 2)
+    {
+        return 1;
+    }
+    Log::initialize(Log::Cout, Log::Warning, argv[0]);
+    
+    std::ifstream file(argv[1]);
+    if(!file.is_open())
+    {
+        return 1;
+    }
+    std::string fileData = std::string((std::istreambuf_iterator<char>(file)), std::istreambuf_iterator<char>());
+    file.close();
+
+    auto msg = SipMessage::make(Data(Data::Share, fileData.data()), true);
```

Comment:
would it be easier to embed the file into the code and have a minimal test package?

Reply:
It make sense, fixed.